gprofiler/main.py

```diff
@@ -306,6 +313,16 @@ def _snapshot(self):
             else:
                 logger.info("Successfully uploaded profiling data to the server")
 
+    def _log_kernel_message_oom(self, message):
```

Comment:
I suppose we could move that into some other object? (`KenelMessagesMonitor`?) so the amount of changes in this main class are lowest

Reply:
moved